.dockerignore

```diff
@@ -1,5 +1,20 @@
-node_modules
-.git
-.github
-.circleci
-.vscode
+#--------------------------
```

Comment:
this change might be problematic for these github actions, like this one: 
https://github.com/dynatrace-oss/barista/blob/master/.github/workflows/cherry-picker.yml
https://github.com/dynatrace-oss/barista/tree/master/tools/cherry-picker

Can we check if they work first?

Reply:
they have their own `.dockerignore` files. So I thought it should be safe but how can we test them?

Reply:
I will add some merge-ready label and wait if the pull request labeller is working. 

Consider the label `merge-ready` as **not merge ready** :D

Reply:
![image](https://user-images.githubusercontent.com/11156362/77318032-98464a80-6d0c-11ea-8ea8-d68a86e9036d.png)

The pull request labeller worked fine and it has the same configuration as the cherry-picked.  So everything is working! 👍